CHANGELOG.rst

```diff
@@ -1,3 +1,11 @@
+Next release
+-------------------
+
+Bugfixes
+========
+
+* bd3d868f Set dofv to nan when candidate_model is None (#1101)
```

Comment:
:warning: Given this is a PR this commit handle might not be the final one.

Reply:
Yes, I see.  I am thinking is there a better way to handle it so that we will not forget to add each fix in the CHAGNELOG? 

> ⚠️ Given this is a PR this commit handle might not be the final one.

Reply:
Put ??? instead?

Reply:
It is enough to have the issue number. If the issue #1101 is mentioned here it will have a connection.